solutions/fully-configurable/variables.tf

```diff
@@ -259,13 +259,13 @@ variable "trusted_profile_roles" {
 
 variable "provision_cbr" {
   type        = bool
-  description = "Whether to enable the creation of context-based restriction rules and zones in the solution. Default is false."
+  description = "Whether to enable the creation of context-based restriction rules and zones in the solution. Default is false. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-cbr/blob/main/modules/fscloud/README.md"
```

Comment:
this is missing the closing bracket, and the full stop at the end of the sentence.